sw/otbn/crypto/p256_ecdsa_sca.s

```diff
@@ -0,0 +1,125 @@
+/* Copyright lowRISC contributors. */
+/* Licensed under the Apache License, Version 2.0, see LICENSE for details. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/**
+ * Elliptic curve P-256 ECDSA
+ *
+ * Uses OTBN ECC P-256 lib to perform an ECDSA operations.
+ */
+
+.section .text.start
+.globl start
+start:
```

Comment:
Is there a meaningful difference between this file and `p256_ecdsa.s`? Same question for P384. If possible, it would be nice to avoid introducing duplicate assembly files.

Reply:
If I understand it correctly, the main difference is that in the `_sca` versions, reads to RND/URND are skipped. Reasons for doing is are:
1. Reduce and stabilize execution time due to RND reads, these are always answered by EDN/CSRNG/Entropy SRC which have quite some latency that may be unpredictable.
2. Switch any masking/blinding off. This is helpful to set up the initial analysis.

I think for now this is fine. We need to start somewhere. Eventually it would be better to not replicate the code but instead have a FPGA-only hardware modification to achieve the same. This is what we do for AES. For KMAC, there exists the possibility to run in a Software Entropy mode where software once reseeds the PRNG and then the PRNG runs forever. This is pretty good for the experiments. I've added an item for this in #13905.

Reply:
Thanks @vogelpi, this is my understanding too based on [this link](https://github.com/newaetech/ot-sca/blob/ecc-analysis/reports/ecc/README.md).

I can keep `_sca.s` files for now, and I can remove them after applying the FPGA changes listed in #13905. @jadephilipoom, Would it be okay for now?

Reply:
Makes sense! Happy to merge the duplicates for now.